.github/workflows/build-scheds.yml

```diff
@@ -0,0 +1,49 @@
+name: build-scheds
+run-name: ${{ github.actor }} PR run
+on: [pull_request]
+jobs:
+  build-schedulers:
+    runs-on: ubuntu-20.04
```

Comment:
I'm wondering if we should target `ubuntu-22.04` (or potentially even `ubuntu-latest`) instead, in this way we don't have to download llvm 17 or any other external package, all we need is in the distro.

Reply:
Another cool thing that we could do is to use `virtme-ng` (https://github.com/arighi/virtme-ng) to actually run the scx schedulers with any kernel that we want. In this case for example, we could `apt download` the latest kernel from my ppa (with the sched-ext support), unpack the kernel, then use `vng -r /path/to/vmlinuz --user root -- scx_simple` and test the scheduler without having to install a custom kernel.

Reply:
> I'm wondering if we should target `ubuntu-22.04` (or potentially even `ubuntu-latest`) instead, in this way we don't have to download llvm 17 or any other external package, all we need is in the distro.

Heh, I'd originally tried to use ubuntu-22.04, but apt-add-repository was hanging for some of the external dependencies. If we can just use everything in the distro itself then that would be great. I'll spin up another PR to try and get it working.

>Another cool thing that we could do is to use virtme-ng (https://github.com/arighi/virtme-ng) to actually run the scx schedulers with any kernel that we want. In this case for example, we could apt download the latest kernel from my ppa (with the sched-ext support), unpack the kernel, then use vng -r /path/to/vmlinuz --user root -- scx_simple and test the scheduler without having to install a custom kernel.

It would be _great_ if we could use virtme-ng. I'd actually like to use that with the sched-ext repo as well, so if we can get it figured out here then I'll also get it setup there.